FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -195,6 +195,33 @@ static uint16_t pusOpenUdpPorts[ democonfigOPEN_UDP_PORTS_ARRAY_SIZE ];
  */
 static Connection_t pxEstablishedConnections[ democonfigESTABLISHED_CONNECTIONS_ARRAY_SIZE ];
 
+
+/**
+ * @brief Custom metrics array.
+ */
+static CustomMetric_t pxCustomMetrics[ democonfigCUSTOM_METRICS_ARRAY_SIZE ];
+
+
+/**
+ * @brief Task status array for uxTaskGetSystemState().
```

Comment:
Readers may not be already aware of `uxTaskGetSysteState` function. This comment can be made more generic about what this array memory is going to store like "status information of tasks running in the system"